### ISSUE TITLE
Xp Cost

### ISSUE THREAD

**Issue by Beez3-A** [Fisch37/waypoints-vanilla#8]
Good Day Fisch37

Thanks for this spectacular mod its Amazing!! Best ever and I way prefer it over waystones!

Would love to know how the XP cost works ? I'm currently in creative testing the mod and i see in the description it 
says cost... 

I've signed a home, named "home" and after I did that I went far away to see if it would cost me any XP, but it never did.

So my question is, is the XP cost in the mod ?  and if so how do I enable it so that when I teleport to a location it costs me an XP level ?

I want this so that it's not so OP 

I have tried using the /function commands but I don't understand. Also I would like to enable it from the mod internaly so I dont have to use cheats in a single player world. 

Thanks!

**Comment by Fisch37** [comment]
I never ended up implemeting waypoint cost the way I wanted, **but** you can still set the waypoint cost for new waypoints with `/scoreboard players set wps_default_cost f37_waypoints_RAM <CUSTOM COST>`

e.g. `/scoreboard players set wps_default_cost f37_waypoints_RAM 2` will cost 2 levels per teleport

**Comment by Beez3-A** [comment]
Thanks for your swift reply

Could this be done inside the mod itself ?

**Comment by Fisch37** [comment]
If you mean a kind of configuration value, then the answer to that is sadly no.

Even though I've packaged this project as a mod on Modrinth (mainly so that it can be loaded across worlds), it's technically a datapack and therefore fully done with commands. Commands can't access files, so I can't store configs there.

I always wanted to do a configuration screen, but then I ran into some issues in the development process so I never ended up doing that
